packages/govuk-frontend-review/src/common/middleware/assets.mjs

```diff
@@ -19,6 +19,9 @@ const frontendPath = packageTypeToPath('govuk-frontend', {
 
 router.use('/assets', express.static(join(frontendPath, 'assets')))
 router.use('/javascripts', express.static(frontendPath))
-router.use('/stylesheets', express.static(join(paths.app, 'dist/stylesheets')))
+router.use('/stylesheets', [
+  express.static(frontendPath),
```

Comment:
**question** Just to confirm that I understand this change properly: this leads Express to also look in `govuk-frontend` (however resolved earlier in the file) alongside looking at the built stylesheets from the review app. In turn, this enables it to find the `/stylesheets/govuk-frontend.min.css` from the boilerplate markup, is that it?

Reply:
Yeah Express.js accepts an array of routes to try in order

Makes sure we can resolve CSS files from both locations:

```
/stylesheets/govuk-frontend.min.css
/stylesheets/app.min.css
```